### ISSUE TITLE
r0.5 - ConfigProto.java is created with incorrect package name for RPCOptions

### ISSUE THREAD

**Issue by lvsubbu** [tensorflow/java#592]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
Linux
IBM S390x Platform
Tensorflow - 2.14.1
TF Java - r0.5
JavaCpp.jar 1.5.8 (build for IBM S390x platform)
Java 11 (OpenJdk) and Maven 3
Tested JavaCpp.jar by sample application
Tested Tensorflow 2.14.1 as well

**Describe the current behavior**

While compiling (javacpp) the presets tensorflow.cpp file - java/tensorflow-core/tensorflow-core-apisrc/gen/java/org/tensorflow/framework/ConfigProto.java is created with an incorrect package name for the field RPCOptions. Where as other fields are created right package name.

```
public final class ConfigProto extends
    com.google.protobuf.GeneratedMessageV3 implements
    // @@protoc_insertion_point(message_implements:tensorflow.ConfigProto)
    ConfigProtoOrBuilder {
/* correct one */
  public static final int GPU_OPTIONS_FIELD_NUMBER = 6;
  private org.tensorflow.framework.GPUOptions gpuOptions_;
/* incorrect one */
  public static final int RPC_OPTIONS_FIELD_NUMBER = 13;
  private tensorflow.RpcOptions.RPCOptions rpcOptions_;
```

How to correct this, I tried changing, config.proto, rpc_options.proto, config.pb.h, rpc_options.pb.h, , also tried to by explicitly setting in presets by .put(new Info("tensorflow::RPCOptions").javaNames("org.tensorflow.framework.RPCOptions")). Nothing works. Please tell me where and what I should fix to resolve this.

1. Can we disable RPCOptions,  ( I tried by commenting it out in .proto, .h, and java), no luck

**Describe the expected behavior**

**Code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by Craigacp** [comment]
The Java bindings for the protos are created by the bazel build not by JavaCPP. We completely refactored how the protos worked when moving to 2.14 (https://github.com/tensorflow/java/commit/72c33b85c4a9991fee9ad9bb797b73078ad8e553#diff-2f8fb84d11020c8bc5459668a7399a6d01643cd415ff6c80501980ea2d65a1b3), due to various proto packaging issues. The linked commit is only the start of it, we had to do a bunch of build system changes as well. It's all ended up bundled together due to the work to remove our build time dependency on bazel.

If you have a hard requirement on TF 2.14 it may be easier to start with TF-Java 1.0.0 and try to downgrade the TF version from 2.16.2 to 2.14.2, as 2.15 and 2.16 are both small releases, whereas the jump from 2.10 to 2.14 is pretty big as TF still had a larger development team at the start of that period.

**Comment by lvsubbu** [comment]
Thanks a lot, @Craigacp , It solved all our issues integrating with TF 2.14.1. I am closing this ticket.

Thanks,
Subramonia.Pillai@ibm.com

**Comment by lvsubbu** [comment]
Thanks a lot, @Craigacp , It solved all our issues integrating with TF 2.14.1. I am closing this ticket.

Thanks,
[Subramonia.Pillai@ibm.com](mailto:Subramonia.Pillai@ibm.com)
